lambda/MAS/Controllers/ContentController.cs

```diff
@@ -26,10 +29,10 @@ public ContentController(IContentService contentService, IS3Service s3Service, I
         }
 
         //PUT api/content/5dac429284dd4afe5eb8fae6
-        [HttpPut("{key}")]
-        public async Task<IActionResult> PutAsync(string key)
+        [HttpPut]
+        public async Task<IActionResult> PutAsync([FromBody] string json)
         {
-            var item = await _contentService.GetItemAsync(key);
+            var item = JsonConvert.DeserializeObject<Item>(json);
```

Comment:
If we're no longer using GetItemAsync() can you get rid of the method in ContentService?